systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -388,6 +379,7 @@ void testAutoReplaceClientsCaKeysTriggeredByAnno() {
                 false);
     }
 
+    // TODO flaky fail from time to time on client timeouts
```

Comment:
As you can see throughout the STs, TODOs are forgettable :) 
You should create an issue and reference it here instead of having just `TODO`